src/pages/home/Home.jsx

```diff
@@ -0,0 +1,107 @@
+  ];
+
+  const data2 = [
+    "https://assets.myntassets.com/f_webp,w_196,c_limit,fl_progressive,dpr_2.0/assets/images/2022/2/6/bfc80f5e-c5c0-4623-9c12-57ac4f203d1d1644167894915-USPA_Kids_min_50_OFF.jpg",
+    "https://assets.myntassets.com/f_webp,w_196,c_limit,fl_progressive,dpr_2.0/assets/images/2022/2/6/2124ec05-5fef-4648-92b7-4b5b27e106e71644167894820-Forever21.jpg",
+    "https://assets.myntassets.com/f_webp,w_196,c_limit,fl_progressive,dpr_2.0/assets/images/2022/2/6/ab530085-ad5a-4e0d-8831-a02b40567e101644167894897-UCB.jpg",
+    "https://assets.myntassets.com/f_webp,w_196,c_limit,fl_progressive,dpr_2.0/assets/images/2022/2/6/df0608cf-36e4-480e-89d5-06887bc4d3eb1644167894835-Louis_Philippe.jpg",
+    "https://assets.myntassets.com/f_webp,w_196,c_limit,fl_progressive,dpr_2.0/assets/images/2022/2/6/56b82797-8b09-43d5-8be6-b4b3bfb001e61644167894934-W.jpg"
+  ];
+
+  const data3 = [
+    "https://assets.myntassets.com/f_webp,w_196,c_limit,fl_progressive,dpr_2.0/assets/images/2022/2/6/ce422810-6997-4d70-926a-f76029b322981644166887490-Bedsheets.jpg",
+    "https://assets.myntassets.com/f_webp,w_196,c_limit,fl_progressive,dpr_2.0/assets/images/2022/2/6/4a879694-c72e-42f5-b57c-f75e41f3a1971644166887497-Blankets_-_Quilts_min_50.jpg",
+    "https://assets.myntassets.com/f_webp,w_196,c_limit,fl_progressive,dpr_2.0/assets/images/2022/2/6/7956f4d9-3dff-4b01-942d-f78c792374251644166887515-Furnishings_-_Curtains-_Cushion_Covers-_Sofa_covers-.jpg",
+    "https://assets.myntassets.com/f_webp,w_196,c_limit,fl_progressive,dpr_2.0/assets/images/2022/2/6/2d014c4f-21fa-45ba-a0dd-c689314553f61644166887506-Floorings_-Carpets-_Runners-_Doormats-.jpg",
+    "https://assets.myntassets.com/f_webp,w_196,c_limit,fl_progressive,dpr_2.0/assets/images/2022/2/6/2e4acf94-a807-4044-b630-b1b4c33d29c31644166887523-Home_Decor_min_70_off.jpg"
+  ];
+
+  return (
+    <>
+      <Navbar />
+
+      <header class="head">
+        <p>
+          <a class="home" href="#">
+            <strong>Home</strong>
+          </a>
+          /
+          <a class="product" href="/pages/product.html">
+            Product
+          </a>
+        </p>
+        <p>
+          <a class="logout" href="/pages/logout.html">
+            Log Out
+          </a>
+        </p>
+      </header>
+
+      <section class="container">
+        <img
+          class="deals"
+          src="https://assets.myntassets.com/f_webp,w_980,c_limit,fl_progressive,dpr_2.0/assets/images/2022/2/10/104a60c9-1018-4071-a85f-09b5a6cd97cf1644486926039-Unbelievable-Deals.jpg"
+          alt="deals.img"
+        />
+        <main class="sub-container">
+          {data1.map((item) => (
+            <>
+              <a href="#">
+                <div class="retina">
+                  <img class="img-size" src={item} alt="card-image" />
+                </div>
+              </a>
+            </>
+          ))}
+        </main>
+
+        <img
```

Comment:
using only images as a component is a bad idea try to make components out of it to make it more consistent.